src/sentry/static/sentry/app/components/events/interfaces/stacktraceLink.tsx

```diff
@@ -124,7 +164,81 @@ class StacktraceLink extends AsyncComponent<Props, State> {
     return null;
   }
   renderNoMatch() {
-    //TODO: Improve UI
+    const {showModal, sourceCodeInput} = this.state;
+    const {organization} = this.props;
+    const filename = this.props.frame.filename;
+    const baseUrl = `/settings/${organization.slug}/integrations`;
+
+    if (this.integrations) {
```

Comment:
@MeredithAnya Remember an empty array is truthy for Javascript, guessing you mean to check if the length is greater than 0, yea?